_layouts/post.html

```diff
@@ -57,3 +57,5 @@ <h1 class="post-title">{{ page.title }}</h1>
   {%- endif -%}
 
 </div>
+
+<script defer src="{{ '/assets/js/copy_code.js' | relative_url }}" type="text/javascript"></script>
```

Comment:
this line should go to `_includes/scripts/misc.html`, which will then be included in the default layout: https://github.com/alshedivat/al-folio/blob/master/_layouts/default.html#L30